shared/utils/domains.test.ts

```diff
@@ -2,133 +2,149 @@ import { stripSubdomain, parseDomain, isCustomSubdomain } from "./domains";
 // test suite is based on subset of parse-domain module we want to support
 // https://github.com/peerigon/parse-domain/blob/master/test/parseDomain.test.js
 describe("#parseDomain", () => {
+  beforeEach(() => {
+    process.env.URL = "https://example.com";
+  });
```

Comment:
These are all the same tests, plus a few more that I added — the functionality of `parseDomain` now relies on the existence of `env.URL` to do the parsing, so I added setup to the beginning of each test